src/pip/_internal/cli/base_command.py

```diff
@@ -162,7 +162,9 @@ def main(self, args):
             )
         elif sys.version_info[:2] == (2, 7):
             message = (
-                "A future version of pip will drop support for Python 2.7."
+                "A future version of pip will drop support for Python 2.7. "
+                "More details about Python 2 support in pip, can be found at "
+                "https://pip.pypa.io/en/latest/development/release-process/#python-2-support."  # noqa
```

Comment:
I would delete the period at the end of the URL so that it will go to the right section if people cut-and-paste the URL into a browser.

Reply:
Good call. :)